app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -40,89 +40,78 @@ let state = {
       sequence: 1
     }
   },
-  txs: [
-    {
-      tx: {
-        hash: "x",
-        inputs: [
-          {
-            denom: "mycoin",
-            amount: 1000
-          },
-          {
-            denom: "fermion",
-            amount: 2300
-          }
-        ]
-      }
-    }
-  ],
   nonces: { [addresses[0]]: 0 },
   txs: [
     {
       tx: {
         value: {
-          msg: {
-            value: {
-              inputs: [
-                {
-                  coins: [
-                    {
-                      denom: "jbcoins",
-                      amount: 1234
-                    }
-                  ],
-                  address: makeAddress()
-                }
-              ],
-              outputs: [
-                {
-                  coins: [
-                    {
-                      denom: "jbcoins",
-                      amount: 1234
-                    }
-                  ],
-                  address: addresses[0]
-                }
-              ]
+          msg: [
+            {
+              value: {
+                inputs: [
+                  {
+                    coins: [
+                      {
+                        denom: "jbcoins",
+                        amount: 1234
+                      }
+                    ],
+                    address: makeAddress()
+                  }
+                ],
+                outputs: [
+                  {
+                    coins: [
+                      {
+                        denom: "jbcoins",
+                        amount: 1234
+                      }
+                    ],
+                    address: addresses[0]
+                  }
+                ]
+              }
             }
-          }
+          ]
         }
       },
+      hash: "999ADECC2DE8C3AC2FD4F45E5E1081747BBE504A",
       height: 1
     },
```

Comment:
worried this tx format might not be in sync w new tm components... might have to update my open PR if tx format is updated again